sdk-usage/.env.example

```diff
@@ -0,0 +1 @@
+DEPLOYER_MNEMONIC=
```

Comment:
Is this even used, and should we keep `.env` variables in the repo?

Reply:
This file serves as a schema for `.env`,  no variables will be kept inside of it